src/components/swap/SwapHeader.tsx

```diff
@@ -13,14 +15,28 @@ const StyledSwapHeader = styled.div`
   color: ${({ theme }) => theme.textSecondary};
 `
 
+const TextHeader = styled.div<{ color: string }>`
+  color: ${({ color }) => color};
+  margin-right: 8px;
+  display: flex;
+  line-height: 20px;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`
+
 export default function SwapHeader({ allowedSlippage }: { allowedSlippage: Percent }) {
+  const theme = useTheme()
+  const fiatOnRampButtonEnabled = useFiatOnRampButtonEnabled()
+
   return (
     <StyledSwapHeader>
       <RowBetween>
-        <RowFixed>
-          <ThemedText.DeprecatedBlack fontWeight={500} fontSize={16} style={{ marginRight: '8px' }}>
+        <RowFixed style={{ gap: '8px' }}>
+          <TextHeader className={subhead} color={theme.textPrimary}>
```

Comment:
if this is the only usage of `TextHeader` you can just add the `theme.textPrimary` color to the styled component directly, same with whatever properties are in `subhead`

Reply:
why not just use `<ThemedText.SubHeader>` as a font wrapper

Reply:
because ThemedText.SubHeader actually doesn't have the right properties for what Figma asks for 😬 i've found NFT's sprinkles to be more consistent / up to date with what we have in Figma. 

Specifically: 
Figma Line Height = 24px = sprinkles subhead line height
Figma Font Weight = 500 = sprinkles font weight

For ThemedText.Subheader, it has: Font weight = 600, and no specification for line height.

Reply:
aren't you setting the line-height to be 20 [here](https://github.com/Uniswap/interface/pull/6240/files#diff-020959a0e0e3d1c593444c25634669a19de67351f4af27bb32ba575469112eefR22)?